content/faqs/How-do-I-uninstall-remove-Privacy-Badger.es.md

```diff
@@ -3,8 +3,10 @@ question: ¿Cómo desinstalo o elimino Privacy Badger?
 weight: 21
 ---
 
-Firefox: Vea la página ["Cómo eliminar extensiones y temas"](https://support.mozilla.org/en-US/kb/disable-or-remove-add-ons#w_how-to-remove-extensions-and-themes) de Soporte de Mozilla.
+Firefox: Consulte la página de ayuda [Desinstalar o eliminar los complementos](https://support.mozilla.org/es/kb/desinstalar-o-eliminar-los-complementos#w_eliminar-extensiones-y-temas) de Mozilla.
 
-Cromo: Consulte la página de ayuda de ["Install and manage extensions"](https://support.google.com/chrome_webstore/answer/2664769?hl=en) Chrome Web Store.
+Cromo: Consulte la página de ayuda [Instalar y administrar extensiones](https://support.google.com/chrome_webstore/answer/2664769?hl=es) de Chrome Web Store.
```

Comment:
We may want to switch to using "Chrome" (instead of "Cromo") in Spanish, as this is what Google itself does in their localized docs.